doc/toc.yml

```
@@ -7,6 +7,6 @@ items:
 - name: Controls
   items:
     - name: CardContentControl
```

maybe change both title and file name to "Cards"?
```suggestion
    - name: Cards
```

---

good catch thanks !